modules/core/src/main/java/org/apache/synapse/aspects/flow/statistics/tracing/opentelemetry/management/TelemetryConstants.java

```diff
@@ -55,6 +55,7 @@ public class TelemetryConstants {
     public static final String DEFAULT_ZIPKIN_HOST = "localhost";
     public static final String DEFAULT_ZIPKIN_PORT = "9411";
     public static final String ZIPKIN_API_CONTEXT = "/api/v2/spans";
+    public static final String OPENTELEMETRY_PROPERTIES = "opentelemetry.otlp.header_properties.";
```

Comment:
Shall we make this `OPENTELEMETRY_PROPERTIES_PREFIX`?

Reply:
Done and resolved